tests/playbooks/test_minimal.yaml

```diff
@@ -71,6 +71,10 @@
       tags:
         - autoscaling_adoption
       when: telemetry_adoption|default(true)
+    - role: manila_adoption
+      tags:
+        - manila_adoption
+      when: manila_backend|default('')
```

Comment:
@tosky an alternative would be to have `manila_backend | default('') | length > 0`, so we can actually run the code if there's a backend set as extra_vars.